Cargo.toml

```diff
@@ -32,6 +32,9 @@ wasm-bindgen-test = "0.3.18"
 [features]
 # Implement std-only traits for getrandom::Error
 std = []
+# Disable `/dev/urandom` fallback for Linux and Android targets.
+# Bumps minimum supported Linux kernel version to 3.17 and Android API level to 23 (Marshmallow).
+disable_urandom_fallback = []
```

Comment:
This feature name is a little confusing, because it is Linux-specific. Enabling this feature won't disable reading from `/dev/urandom` on other non-Linux targets.

Ideally, we would just have a `disable_fallback` feature which could be enabled on whatever targets/arches people wanted. However, doing this is complicated (https://github.com/rust-lang/cargo/issues/1197), so I'm not sure how well this approach would work.

Given this the two alternatives would be:
  - Having a `disable_legacy_fallback` which disables all sufficiently old fallback methods.   For example, this would also disable the fallback code on Solaris / Illumos
  - Rename the feature to `linux_disable_file_fallback` to make it clear that this is a feature-specific target.

We could also just remove this feature from this PR and have a followup PR that just adds the fallback-disabling

Reply:
`linux_disable_file_fallback` sounds good. It's a bit long, but should be tolerable enough.

Reply:
`linux_disable_fallback` would also be fine. On linux, there's only one relevant fallback (reading from a file).